js/local-storage.js

```diff
@@ -0,0 +1,25 @@
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
+
+function saveFormData() {
+  const formData = {
+    name: nameInput.value,
+    email: emailInput.value,
+    message: messageInput.value,
+  };
+  localStorage.setItem('formData', JSON.stringify(formData));
+}
+
+function loadFormData() {
+  const formData = JSON.parse(localStorage.getItem('formData')) || {};
+  nameInput.value = formData.name || '';
+  emailInput.value = formData.email || '';
+  messageInput.value = formData.message || '';
```

Comment:
you'll have to update this too eventually

Reply:
Please check the above comment, it's related to this too